models/app.models.js

```diff
@@ -18,7 +18,16 @@ function getArticleIdData(id) {
         }
         return rows
     })
-    
 }
 
-module.exports = {getTopicData, getArticleIdData}
\ No newline at end of file
+function getArticleData() {
+    return db.query(`SELECT articles.*, (SELECT COUNT(*) FROM comments WHERE comments.article_id = articles.article_id) AS comment_count FROM articles`).then(({rows}) => {
+        rows.forEach((article) => {
+            article.comment_count = Number(article.comment_count)
+            delete article.body
+        })
+        return rows
+    })
+}
```

Comment:
I like that you've deleted the body, however, this could and should be done by selecting the relevant columns in the sql query rather than selecting all.

I've also noticed that you've used a sub query here to select the comments, this is fine and it works but it would be better for you to show your understanding of a JOIN.